server/the-usher-openapi-spec.yaml

```diff
@@ -758,6 +799,33 @@ components:
         - client_id: client-app2
           clientname: Client Application 2
 
+# PERSONAS
+#---------------------
+    Persona:
+      type: object
+      properties:
+        key:
+          type: integer
+          minimum: 1
+          format: int32
+        tenantkey:
+          type: integer
+          minimum: 1
+          format: int32
+        user_context:
+          type: string
+          nullable: true
+        sub_claim:
+          type: string
+        created_at:
+          type: string
+        updated_at:
```

Comment:
@k1ch  are we always returning these "db" fields? I feel the `created_at` probably is fine, not sure if we need to return the updated_at?

Reply:
Sure, I remove `updated_at` from the returning list. I keep is in OAS though since we might want to return this field for `GET:tenant/{tennat_key}/personas`